src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputTryCatchIfElse2.java

```diff
@@ -13,16 +13,13 @@ public static void main(String[] args) {
     try {
       /* foo */
     } catch (NullPointerException e) {
-      // violation below 'Empty catch block.'
+      /* foo */
     } catch (Exception e) {
+      /* foo */
     } finally {
       after = true;
     }
 
-    if (after) {
-      System.out.println("after");
-    } else {}
```

Comment:
this was a repetitive case, similar one is present at InputTryCatchIfElse.java

https://github.com/checkstyle/checkstyle/blob/4f889899caafb691ff4baad43e378c6879392cec/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputTryCatchIfElse.java#L12-L13